greedy_methods/interval_scheduling.py

```diff
@@ -0,0 +1,27 @@
+# Interval Scheduling: https://en.wikipedia.org/wiki/Interval_scheduling
```

Comment:
"Interval scheduling" is the name of the problem, not the algorithm. What is the name of the algorithm?

Reply:
If this is going to land, it should be with the other scheduling algorithms...
https://github.com/TheAlgorithms/Python/tree/master/scheduling